src/main/resources/i18n/displayStrings.properties

```diff
@@ -1293,7 +1293,7 @@ Please select one of the existing MultiSig transactions from your wallet which w
 deposit transaction used in the failed trade.\n\n\
 You can find the correct transaction by opening the trade details window (click on the trade ID in the list)\
  and following the trading fee payment transaction output to the next transaction where you see \
-the Multisig deposit transaction (the address starts with 3). That transaction ID should be \
+the MultiSig deposit transaction (the address starts with 3). That transaction ID should be \
```

Comment:
Can you provide resources as to why it should be capitalized this way? My quick searching on the matter showed me that "Multisig" (lower case 's') is the more conventional form.

Reply:
I made a search of all "multisig" instances in the file and most of them where capitalized. So I also capitalized the rest...

I could change to lowercase if it's preferred.

Reply:
While fixing the bisq-docs repo, I saw multiple instances of "MultiSig", "Multisig" and "multisig".

So let's agree on one and I could change all the rest :)

Reply:
There's also no reason in principle to capitalize multisig. It's not a proper noun, it's an abbreviation of the compound adjective "multi-signature". I don't think anyone would suggest that the latter should be spelled "Multi-Signature" or even "Multi-signature", so there's no reason that "multisig" should be either.

Sometimes this sort of thing happens when a concept is new, just to set it off  or call it out as important, but these unjustified capitalizations usually fade away, and it seems they already have in the case of "multisig".